### ISSUE TITLE
wgPatentPolicy and CG?

### ISSUE THREAD

**Issue by tidoust** [w3c/respec#3420]
## Important info

- URL to affected spec: https://w3c.github.io/cogai
- ReSpec version: 26.4.1

## Description of problem

**What happened?:**
ReSpec reports the following error:
```
(w3c/headers): Invalid wgPatentPolicy value: "undefined". Please use one of: PP2017, PP2020 .
```

**Expected behavior:**
The [doc](https://respec.org/docs/#wgPatentPolicy) says that the setting is being taken care of when `group` is used. That is the case here (`group: "cogai"`).

(On top of this, I believe that this setting does not affect CG drafts)

**Comment by sidvishnoi** [comment]
Regression from https://github.com/w3c/respec/pull/3411
https://github.com/w3c/respec/blob/d66c31e9ac1d34e618987645d29cec6ff015fcc6/src/w3c/headers.js#L729
should instead be `if (!conf.wgPatentPolicy) return;`

**Comment by gkellogg** [comment]
It looks like this commit was included in 26.4.1, but I'm still getting the error on https://json-ld.github.io/json-ld-star/ and https://w3c.github.io/rdf-star/cg-spec/editors_draft.html.

**Comment by sidvishnoi** [comment]
This will be available in next release tomorrow: v24.5.0
